flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/contrib/streaming/state/RocksDBRocksStateKeysAndNamespacesIteratorTest.java

```diff
@@ -129,4 +131,74 @@ public void testIterator() throws Exception {
             }
         }
     }
+
+    @Test
+    public void testIteratorUpdateMapStateMultipleTimes() throws Exception {
```

Comment:
First of all, this test has many code same as `#testIteratorHelper`. Secondly, I think we should add new test in `StateBackendTestBase` to verify all state-backends.

Reply:
Very Thanks for reviewing. I total agree with your opinion and I've moved the test to `StateBackendTestBase`.
BTW , I think it is unnecessary to add a new test but just need to add some MapState-related case in `#testGetKeysAndNamespaces`. what do you think ?